typing/rec_check.ml

```diff
@@ -787,6 +788,12 @@ let rec expression : Typedtree.expression -> term_judg =
           Delay
       in
       expression e << lazy_mode
+    | Texp_letop{let_; ands; body; _} ->
+        let case_env c m = fst (case c m) in
+        join [
+          list binding_op (let_ :: ands) << Dereference;
+          case_env body << Delay
+        ]
```

Comment:
IIUC you're ignoring the the rhs of the equal sign of all the bindings. Which seems weird (and definitely different from what `Texp_let` does).
Is that intended?

Reply:
In `binding_op` it does both the operation and the rhs.